### PR TITLE
ADD:SignIN and SignUp Pages

### PR DESCRIPTION
**Pull request by BIPS8812**

### CODE REVIEW

File: src/Components/SignInForm/SignInForm.module.scss [hunk 1]
```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap");
+
```
Comment (codadept) [inline src/Components/SignInForm/SignInForm.module.scss]: Import the font in index.scss and create a css variable and then use it in required files

File: src/Components/SignUpForm/SignUpForm.module.scss [hunk 2]
```diff
@@ -0,0 +1,156 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap");
+
```
Comment (codadept) [inline src/Components/SignUpForm/SignUpForm.module.scss]: See you have imported the font twice. REDUNDANT. Use the suggested approach

File: src/Components/SignUpForm/SignUpForm.module.scss [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      border: 0.5px solid #b3b3b3;
+      border-radius: 6px;
+      background: #f8f8f8;
+
+      &:focus {
+        outline: none;
+      }
+
+      &::placeholder {
+        font-family: "Poppins";
```
Comment (codadept) [inline src/Components/SignUpForm/SignUpForm.module.scss]: Use the variable defined in index.scss. Always use variables. Check for all the scss files you have written

File: src/Pages/SignIn/SignIn.module.scss [hunk 4]
```diff
@@ -0,0 +1,150 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@500&display=swap');
+
```
Comment (codadept) [inline src/Pages/SignIn/SignIn.module.scss]: Fix this too

File: src/Pages/SignUp/SignUp.module.scss [hunk 5]
```diff
@@ -0,0 +1,142 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@500&display=swap');
```
Comment (codadept) [inline src/Pages/SignUp/SignUp.module.scss]: This too

**Review by codadept — CHANGES_REQUESTED** [review]

File: src/Pages/SignUp/SignUp.jsx [hunk 6]
```diff
@@ -8,10 +8,6 @@ const SignUp = () => {
       <div className={style.SignUp}>
         <SignUpForm />
       </div>
-      <div className={style.gdsc}>
```
Comment (JDeepD) [inline src/Pages/SignUp/SignUp.jsx]: Since Footer is now present, This portion is no longer required.

**Comment by JDeepD** [comment]
The Navbar is present and still has all the nav-items that should be accessible to only authenticated user. This will be removed in the Auth Integration PR

**Pull request merged**